README.md

```diff
@@ -41,44 +67,47 @@ Include a diagram describing the application
 - This model uses [MODEL 1] and [MODEL 2] which may not be available in all Azure regions. Check for [up-to-date region availability](https://learn.microsoft.com/azure/ai-services/openai/concepts/models#standard-deployment-model-availability) and select a region during deployment accordingly
     - We recommend using [SUGGESTED REGION]
 - ...
- 
-### Installation
- 
+
+#### Installation
+
 (ideally very short)
  
 - npm install [package name]
 - mvn install
 - ...
- 
-### Quickstart
+
+#### Quickstart
+
 (Add steps to get up and running quickly)
  
 1. Clone the repository and intialize the project: `azd init [name-of-repo]`
```

Comment:
Doesnt azd init need -t ? Or does it default to assuming name of repo?

Reply:
I find this step unclear, since I might take it to mean that I need to first clone the repository and then run azd init, which would be redundant. I'd find it clearer if it said something like: 

```
Bring down the project code:

azd init -t project-name

This will perform a git clone for you.
```

I personally avoid azd init and just tell folks to git clone.